app/views/candidate_interface/continuous_applications_choices/index.html.erb

```diff
@@ -6,25 +6,46 @@
       <%= t('page_titles.continuous_applications.your_applications') %>
     </h1>
 
-    <% unless @application_form_presenter.can_add_more_choices? %>
-      <p class="govuk-body">You cannot add any more applications.</p>
-      <p class="govuk-body">If one of your applications is unsuccessful, or you withdraw or remove it, you will be able to add another application.<p>
-          <p class="govuk-body">
-            <%= govuk_link_to 'Read how the application process works', candidate_interface_guidance_path %>
-          </p>
+    <% if CycleTimetable.today_is_between_apply_1_deadline_and_find_reopens  %>
```

Comment:
do we need a `continuous_applications?` check here too?

Reply:
No because this view is continuous applications only so will be redundant :)